test/integration/clientset_test.go

```diff
@@ -113,8 +113,12 @@ func TestEtcdHealthCheckerSuccess(t *testing.T) {
 	}
 	c := &http.Client{Transport: tr}
 	resp, err := c.Get(clientconfig.Host + "/healthz")
-	if nil != err || http.StatusOK != resp.StatusCode {
-		t.Fatal("health check endpoint should not have failed")
+	if nil != err {
```

Comment:
separate PR plz

Reply:
Debugging improvement. Why split it out?

Reply:
You can split it out if you want it to land sooner.  Your choice.